materialized_aggregations.mdx

```diff
@@ -0,0 +1,124 @@
+
+Chalk aligns buckets on Unix epoch (ignoring leap seconds).
+When serving windowed queries, Chalk uses all buckets containing any overlap with the requested time window.
+
+## How do I use materialized aggregations with Chalk?
+
+Materialize a feature aggregation in Chalk, by supplying the [`materialization`](/api-docs#windowed.materialization) parameter of the windowed function of the feature you want to aggregate.
+ In the example above, we specify a `bucket_duration` of `"1d"`.
+
+The materialization parameter takes in a dictionary that adheres to the schema outlined by MaterializationWindowConfig:
+
+- `bucket_duration`: [Duration](https://docs.chalk.ai/api-docs#Duration)
+
+Duration of each bucket in the window e.g. "1h"
+
+- `bucket_durations`: Mapping\[[Duration](https://docs.chalk.ai/api-docs#Duration), Sequence[[Duration](https://docs.chalk.ai/api-docs#Duration)] | [Duration](https://docs.chalk.ai/api-docs#Duration)]\]
+
+A dictionary that specifically maps bucket_durations to your window intervals e.g. if you wanted to use `"10m"` bucket durations for your 1d window interval and `"3d"` bucket durations for your `"30d"` window interval.
+We recommend explicitly mapping bucket durations to each window if your window intervals have a wide discrepancy e.g. ten minute bucket duration with a one year window interval.
+
+Any window intervals not explicitly included in the `bucket_durations` dictionary will use your supplied `bucket_duration` by default.
+
+`backfill_schedule`: \[[CronTab](https://docs.chalk.ai/api-docs#CronTab) | None\]
+
+The schedule on which to automatically backfill the aggregation.
+For example, `"* * * * *"` or `"1h"`.
+
+`continuous_buffer_duration`: \[[Duration](https://docs.chalk.ai/api-docs#Duration)\] | None
+
+The minimum period of time for which to sample data directly via online query, rather than from the backfilled aggregations.
+
+## Continuous backfills
+
+You can integrate fresh data into your windowed features using continuous backfills.
+
+First, set a schedule to determine when and how frequently backfills should occur.
+In the following example, the `total_transaction_amount` features will be backfilled every day.
+
+```python
+from chalk.streams import Windowed, windowed
+
+@features
+class User:
+    id: int
+    transactions: DataFrame[transactions]
+    total_transaction_amount: Windowed[int] = windowed(
+        "10d",
+        "90d",
+        materialization={
+            "bucket_duration": "1d",
+            "backfill_schedule": "0 0 * * *",
+            "continuous_buffer_duration": "36h",
+        },
+        expression=_.transactions[_.amount].sum(),
+    )
+```
+
+Second, set a `continuous_buffer_duration` to indicate the time window of data to aggregate and compute directly from your online resolvers.
+The rest of the window is computed from the pre-aggregated buckets.
+
+### Managing aggregations
```

Comment:
here we should flag that you can backfill materialized aggs a few ways: 
- using the backfill_schedule on the materialization to run the necessary resolvers
-  streaming (and link to streaming)